build/buildConfig.js

```diff
@@ -35,6 +36,10 @@ module.exports = (bundles, themeEntries, paths, extractThemesPlugin, prod, publi
         'webpack-dev-server': 'webpack-dev-server/client?http://0.0.0.0:8081', // WebpackDevServer host and port
         'webpack': 'webpack/hot/only-dev-server' // "only" prevents reload on syntax errors
     }, bundles, themeEntries),
+    mode: "development",
```

Comment:
Hi, great work, i think this will need to be overridden in prod-webpack.config

Reply:
I changed buildConfig.js to switch using the prod flag